experiment/kind-nftables-e2e.sh

```diff
@@ -0,0 +1,301 @@
+#!/bin/sh
+# Copyright 2018 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# hack script for running a kind e2e
```

Comment:
This seems to be a fork of an older fork of kind's `e2e-k8s.sh`? Should it be re-synced from the original?

Reply:
no need since this is custom for nftables proxy, once nftables graduates to beta we can add to kind API and we can remove this